src/app/buttons/button-loading.ts

```diff
@@ -4,35 +4,45 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import {Component} from "@angular/core";
+import {ClrLoadingState} from "../../clr-angular/utils/loading";
```

Comment:
The app should always import from `"../../clr-angular"`, since it acts as a consumer of Clarity and anything deeper is not public.

Reply:
👍